clans/src/main/java/me/mykindos/betterpvp/clans/progression/perks/BaseFishingPerk.java

```diff
@@ -88,5 +92,6 @@ public void onFish(PlayerStartFishingEvent event) {
     @Override
     public void loadConfig(@NotNull ExtendedYamlConfiguration config) {
         this.requiredLevel = config.getOrSaveObject("fishing.base-fishing-perk.level", 500, Integer.class);
+        this.enabled = config.getOrSaveBoolean("fishing.base-fishing-perk.enabled", true);
```

Comment:
Is there any particular reason why you did this one via loadConfig, and then the rest via the config injector?

Reply:
This one was done by Rey via ConfigAccessor, which I completely missed until I put it in here. I probably should do everything the same way, it makes way more sense to do it via ConfigAccessor.